test/integration/scheduler_test.go

```diff
@@ -103,50 +113,171 @@ func podScheduled(c *client.Client, podNamespace, podName string) wait.Condition
 	}
 }
 
-func DoTestUnschedulableNodes(t *testing.T, client *client.Client) {
-	node := &api.Node{
-		ObjectMeta: api.ObjectMeta{Name: "node"},
-		Spec:       api.NodeSpec{Unschedulable: true},
+// Wait till the passFunc confirms that the object it expects to see is in the store.
+// Used to observe reflected events.
+func waitForReflection(s cache.Store, key string, passFunc func(n interface{}) bool) error {
+	return wait.Poll(time.Second, time.Second*20, func() (bool, error) {
```

Comment:
I can bring it down, copied the polling interval from the scheduler test but it probably isn't. Most cases on localhost it's done on the first poll.